test/manipulation.test.js

```diff
@@ -658,10 +658,63 @@ describe('manipulation', function() {
   });
 
   describe('updateOrCreate', function() {
+    var db = getSchema();
+    var Todo;
+
+    before(function setUpDatabase(done) {
+      Todo = db.define('Todo', {
+        content: String,
+      });
+      done();
```

Comment:
I think you should still call `db.automigrate(done)` to ensure the tables are created? Note that these tests are run by all connectors, and especially SQL connectors need to create tables before you can store any data. 

The way how the code is written now, you are relying on some other tests running earlier to create a table for `Todo` model, possibly with a different set of properties than your Todo model has.